.travis.yml

```diff
@@ -13,6 +13,10 @@ before_install:
 install:
   - npm install
 
+cache:
+  directories:
+    - "node_modules"
```

Comment:
Will the cache invalidate when package-lock changes? Never knew you could do this in travis :tada:

Just curious 🙏

Reply:
> Will the cache invalidate when package-lock changes?

Yes, see https://docs.travis-ci.com/user/caching/

Reply:
Oh that's quite neat!